sycl/include/sycl/ext/oneapi/properties/property.hpp

```diff
@@ -225,8 +225,11 @@ enum PropKind : uint32_t {
   Unaliased = 80,
   EventMode = 81,
   NativeLocalBlockIO = 82,
+  InitialThreshold = 83,
+  MaximumSize = 84,
+  ZeroInit = 86,
   // PropKindSize must always be the last value.
-  PropKindSize = 83,
+  PropKindSize = 87,
```

Comment:
Should these values be updated? I'm presuming `Readonly` was `85`.

Reply:
Fixed, Thanks for noting that as I missed it